src/vs/workbench/contrib/notebook/browser/view/renderers/webviewPreloads.ts

```diff
@@ -159,10 +159,36 @@ async function webviewPreloads(ctx: PreloadContext) {
 				}
 			};
 		};
+	function getOutputContainer(event: FocusEvent | MouseEvent) {
+		for (const node of event.composedPath()) {
+			if (node instanceof HTMLElement && node.classList.contains('output')) {
+				return {
+					id: node.id
+				};
+			}
+		}
+		return;
+	}
+	let lastFocusedOutput: { id: string } | undefined = undefined;
+	const handleOutputFocusOut = (event: FocusEvent) => {
```

Comment:
I noticed that around 2-3 years ago, there was a focus tracker for output items.
Code has changed since then, didn'-t want to add a similar concept, as we do not want to track individual items, 
If one output is focused, then rest are automatically un-focused.

Do not want to merge this late in the iteration, small and simple, but would like to get a better understanding of the code and solutions, i.e. would first like to chat about this.

@rebornix /cc